### PR TITLE
[Plans] faq content update

### PR DESCRIPTION
**Pull request by sorta**
Part of https://github.com/travis-ci/travis-web/pull/2068

This is in response to feedback about the content of the FAQ section

### CODE REVIEW

**Comment by sorta** [comment]
Small updates mainly to FAQ section content. Should be ready for review when you get a chance @artursmirnov. Thanks!

File: app/templates/components/ui-kit/text.hbs [hunk 1]
```diff
@@ -1,6 +1,4 @@
-{{#let (element this.tag) as |Tag|}}
-  <Tag
-    class={{this.allClasses}}
-    ...attributes
-  >{{yield}}</Tag>
-{{/let}}
\ No newline at end of file
+{{#let (element this.tag) as |Tag|}}<Tag
```
Comment (artursmirnov) [inline app/templates/components/ui-kit/text.hbs]: There should be a way to handle these whitespace issues. Not now, but eventually. Something like setting container `font-size: 0` and then specify it for `UiKit::Text` nodes directly.

Comment (sorta) [reply app/templates/components/ui-kit/text.hbs]: Yeah totally. I mean, I guess we could put a base style of `font-size: 0` on `body`. Although then absolutely any text that we want to show would need to be in a `UiKit::Text` component. But that's kind of the plan anyway, but then maybe there would be some weird other edge cases or unintended consequences... But yeah, we can definitely think of and work on a way to deal with whitespace.

**Comment by sorta** [comment]
Awesome, thanks for the review! :smile:

File: app/templates/components/ui-kit/text.hbs (hunk 1 above)
Comment (artursmirnov) [reply app/templates/components/ui-kit/text.hbs]: I'd set some sensible font size for body to keep things like base font size and `em` work properly and then set it to `0` on some root `div` for the app. Though, I'm not really sure about this solution, makes sense to google about it :) There should be a battle tested solution :)

Comment (sorta) [reply app/templates/components/ui-kit/text.hbs]: Yeah honestly I'm not really sure about that solution as well haha. It's definitely worth researching before doing anything drastic

Comment (sorta) [reply app/templates/components/ui-kit/text.hbs]: Huh, TIL about Handlebars `~`. This seems to work:
```
{{#let (element this.tag) as |Tag|~}}
  <Tag
  class={{this.allClasses}}
  ...attributes
  >
    {{~yield~}}
  </Tag>
{{~/let}}
```

Comment (sorta) [reply app/templates/components/ui-kit/text.hbs]: Docs about it: https://handlebarsjs.com/expressions.html#whitespace-control

Comment (artursmirnov) [reply app/templates/components/ui-kit/text.hbs]: Wow, TIL 😍 Thanks for the research! I knew it would be simple 😄

**Pull request merged**